data/json/items/chemicals_and_resources.json

```diff
@@ -890,5 +890,24 @@
     "weight": 212,
     "volume": 1,
     "to_hit": -2
-    }
+    },{
+    "type" : "COMESTIBLE",
```

Comment:
If we had water stills we could put it in there to produce drinkable water. Maybe run it through a composting process to produce fertilizer. Other than that I'm drawing a blank at the moment.